src/http/nodegen/middleware/handleHttpException.ts

```diff
@@ -4,19 +4,38 @@ import { HttpException } from '../errors';
 import NodegenRequest from '../interfaces/NodegenRequest';
 import config from '@/config';
 
+export interface HttpExceptionOptions {
+  errorHook?: (error: any) => void,
+  errorLogger?: (error: any) => void,
+}
+
 /**
  * Http Exception handler
  */
-export default () => (err: HttpException, req: NodegenRequest, res: express.Response, next: express.NextFunction) => {
-  if (!(err instanceof HttpException)) {
-    err = createHttpExceptionFromErr(err);
+export default (options: HttpExceptionOptions = {}) => {
```

Comment:
same comment as above - default is fine, but safe navigator still needs to be used